packages/libs/sdk-mixins/src/mixins/descopeUiMixin/constants.ts

```diff
@@ -7,10 +7,10 @@ export const UI_COMPONENTS_URL_KEY = 'base.ui.components.url';
 
 export const UI_COMPONENTS_URL =
   (IS_LOCAL_STORAGE && localStorage.getItem(UI_COMPONENTS_URL_KEY)) ||
-  'https://static.descope.com/npm/@descope/web-components-ui@<version>/dist/umd/index.js';
+  'https://dscpcdn.net/npm/@descope/web-components-ui@<version>/dist/umd/index.js';
```

Comment:
@omercnet why this is the main url? I thought that this should be the fallback (it definitely seems, to me at least, like a fallback / .net, the domain "looks" less reliable than descope

But maybe I am missing something 

Also, if customer now had csp that allows descope.com, won't it break something? Or at least they will have an error in the console log?

Reply:
There's a fallback to old primary url so if for some reason the new fails it still uses the previous one in a non-breaking manner

Re testing, am I wrong to assume to the e2e cover this?